source/2020-10-19-ember-3-22-released.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Ember 3.22 Released
+author: Jared Galanis, Isaac Lee, Ricardo Mendes
+tags: Releases, 2020, 3, 3.22, Version 3.x
+responsive: true
+---
+
+<!-- alex disable retext-equality -->
+
+Today the Ember project is releasing version 3.22 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 3.23 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.22
+
+Ember.js 3.22 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+Ember.js 3.22 introduced several bug fixes, including:
+
+- [#19094](https://github.com/emberjs/ember.js/pull/19094) `RouterService#isActive` now updates tracked properties correctly by consuming `currentURL` and `currentRouteName`;
+- [#19170](https://github.com/emberjs/ember.js/pull/19170) modifier manager 3.22 now accepts the resolved value of `owner.factoryFor` directly, which is in keeping with the way helpers and component managers work; and
```

Comment:
Not sure it matters to call out, but the 3.22 capability for modifier managers also allows modifiers to only re-run based on tracked state that they themselves consume. Previously, the framework would eagerly consume all of the arguments.

Reply:
Thanks for pointing this out. I think we should mention it in the blog post.